File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.cpp
#include <string>
#include <vector>
#include <math.h>
using namespace std;

int solution(int n) {
    int answer = 0;
    vector<int> num;
    while(n!=0)
    {
        num.push_back(n%3);
        n/=3;
    }
    int len = 0;
    for(auto it = num.rbegin();it!=num.rend();it++)
    {
        answer += *it * (int)pow(3,len);
        len++;
    }
    
    return answer;
}

File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.cs
using System;

public class Solution {
    public int solution(int n) {
        int answer = 0;
        string num = "";
        while(n > 0)
        {
            num+=n%3;
            n/=3;
        }
        int len = num.Length-1;
        foreach(char c in num)
        {
            int q = (c-'0');
            answer += (int)Math.Pow(3,len) * q;
            len--;
        }
        return answer;
    }
}